packages/webdriver/src/cjs/index.ts

```diff
@@ -32,20 +39,19 @@ class WebDriver {
      * @param   {object} instance  the object we get from a new browser session.
      * @returns {string}           the new session id of the browser
      */
-    static async reloadSession(instance: Client) {
+    static async reloadSession (instance: Client) {
         const WebDriver = (await import('../index.js')).default
         return WebDriver.reloadSession(instance)
     }
 
-    static get WebDriver() {
+    static get WebDriver () {
         return WebDriver
     }
-}
 
-module.exports = WebDriver
-exports.command = (method: string, encodeUri: string, commandInfo: any, doubleEncodeVariables = false) => {
-    return async function protocolCommand(this: unknown, ...args: unknown[]) {
-        const commandESM = await import('../command.js')
-        return commandESM.default(method, encodeUri, commandInfo, doubleEncodeVariables).apply(this, args)
+    static get command () {
+        return command
     }
 }
+
+module.exports = WebDriver
+exports.command = command
```

Comment:
```suggestion
module.exports.command = command
```

Reply:
`module.exports = WebDriver` which has `command` as static property, so it should be the same, right?

Reply:
the test seem to pass:

```js
const WebDriver = require('../../packages/webdriver')
const command = require('../../packages/webdriver').command

assert.equal(typeof WebDriver.command, 'function')
assert.equal(typeof command, 'function')
```

Reply:
oh I didn't see the refactor above, this was a suggestion based on the initial commit. let me try.

Reply:
yep this should work now

Reply:
👍